tests/test_solvers_fairflow.py

```diff
@@ -449,10 +449,10 @@ def test_solver_fairflow_respect_minimums_2():
     assert solver.solved
     # make sure every reviewer has at least 1 paper
     nrows, ncols = res.shape if len(res.shape) == 2 else (0, 0)
-    for rix in range(nrows):
+    for rix in range(ncols):
         reviewer_count_reviews = 0
-        for pix in range(ncols):
-            if res[rix, pix] != 0:
+        for pix in range(nrows):
+            if res[pix, rix] != 0:
```

Comment:
All 3 of these identical changes are because semantically, the returned matrix from the solver has papers in rows and reviewers in columns. So pix (I assume standing for paper index) should index the rows and rix (reviewer index) should index the columns. It doesn't change any logic but I think it's more semantically accurate.